docs/UserGuide.adoc

```diff
@@ -17,100 +17,139 @@ endif::[]
 By: `AY1920S2-CS2103T-W12-4`      Since: `Feb 2020`      Licence: `MIT`
 
 == Introduction
-**CookBuddy** is for users who prefer a desktop application for managing recipes. More importantly, CookBuddy is optimized for those who prefer to work with a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, CookBuddy can get your recipe management tasks done faster than traditional GUI apps. Interested? Proceed to [Quick Start](#quick-start) to get started. Enjoy!
+
+This user guide provides documentation on CookBuddy such as a quick start guide and an overview of its features.
+
+*CookBuddy* is for users who prefer a desktop application for managing recipes.
+More importantly, *CookBuddy* is optimized for those who prefer to work with a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI).
+
+If you are a fast typist, *CookBuddy* can manage your recipes faster than applications that use a GUI.
```

Comment:
```suggestion
If you are a fast typist, *CookBuddy* can manage your recipes faster than applications than a traditional GUI.
```

Reply:
it was that before, but i changed it since there's no such thing as a "traditional" GUI haha

Reply:
true that!